packages/config/config/devices/0x001d/dz1kd.json

```diff
@@ -16,71 +16,36 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Fade On Time",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 253,
-			"defaultValue": 2,
-			"unsigned": true
+			"$import": "templates/config_template.json#fade_on_time"
 		},
 		{
 			"#": "2",
-			"label": "Fade Off Time",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 253,
-			"defaultValue": 2,
-			"unsigned": true
+			"$import": "templates/config_template.json#fade_off_time"
 		},
 		{
 			"#": "3",
-			"label": "Minimum Light Level",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 100,
+			"$import": "~/templates/master_template.json#minimum_dim_level_1-99",
 			"defaultValue": 10
 		},
 		{
 			"#": "4",
-			"label": "Maximum Light Level",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 100,
-			"defaultValue": 100
+			"$import": "~/templates/master_template.json#maximum_dim_level_0-100"
 		},
```

Comment:
These stand out a bit, since Z-Wave usually has levels of 0/1...99. Do the devices actually accept 100 as a value? Is it in % then?